themes/nekonight_moon/nekonight_moon.theme.sh

```diff
@@ -0,0 +1,18 @@
+#! bash oh-my-bash.module
+# nekonight-moon Bash prompt with source control management
+# Author: Bruno Ciccarino <brunociccarinoo@gmail.com>
+#
+# Theme inspired by:
+#  - Bash_it cupcake theme
+# Demo:
+# ╭─🌙 virtualenv 🌙user at 🌙host in 🌙directory on (🌙branch {1} ↑1 ↓1 +1 •1 ⌀1 ✗)
+# ╰λ cd ~/path/to/your-directory
+
+_omb_theme_nekonight_icon_emoji="🌙"
+
```

Comment:
This blank line only exist in `nekonight_moon.theme.sh`. Could you please remove this line, or could you add a blank line in the corresponding position in `nekonight.theme.sh`?

Reply:
No, sorry. Please dismiss this comment. I overlooked the blank line in `nekonight.theme.sh`.